src/plugins/ssh/index.ts

```diff
@@ -182,6 +182,17 @@ async function spawnSshNode(
 ): Promise<number | null> {
   return new Promise((resolve, reject) => {
     const provider = SSH_PROVIDERS[options.provider];
+
+    const attemptsRemaining = options.attemptsRemaining;
+    if (options.debug) {
```

Comment:
Why did this move? Do we always want to print this out even if access already propagated (probably unlikely that it would propagate immediately)? 🤔

Reply:
It's only printed in debug mode. I found it useful to capture the first attempt if for some reason we never get to the second.